Formula/libpsl.rb

```diff
@@ -0,0 +1,25 @@
+class Libpsl < Formula
+  desc "C library for the Publix Suffix List"
+  homepage "https://github.com/rockdaboot/libpsl"
+  url "https://github.com/rockdaboot/libpsl/releases/download/libpsl-0.14.0/libpsl-0.14.0.tar.gz"
+  sha256 "e8c794bcedf45c87dee4810570bc62ba7130217887e34e164e3eadc2259f2efb"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "md5sha1sum" => :build
+  depends_on :python2 => :build
```

Comment:
Thanks for the tip, @dunn. The build-time script `src/psl-make-dafsa` has the shebang line `#!/usr/bin/env python2`, uses `with` but doesn't need to import it from `__future__`, and doesn't use set literals or set/dict comprehensions, so I think it'll run on either 2.6 or 2.7.